src/schema/devcards.ts

```diff
@@ -200,7 +200,7 @@ export const resolvers: IResolvers<any, Context> = {
         imageUrl: `${process.env.URL_PREFIX}/devcards/${devCard.id.replace(
           /-/g,
           '',
-        )}.png?r=${randomStr}`,
+        )}.svg?r=${randomStr}`,
```

Comment:
We might need to turn the response to SVG so that when the user copies the link, they will have the SVG at hand.

Reply:
I think it's much clearer to return png here and rely on HTML generation for the opengraph link.